src/app/Enums/Role.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Enums;
+
+use MyCLabs\Enum\Enum;
+
+/**
+ * Role Enum
+ */
+class Role extends Enum
+{
+    // 管理者
+    const ADMIN = 'admin';
+    // 一般
+    const MEMBER = 'member';
+
+    /**
+     * 権限名の全件取得
+     */
+    public static function getRoleValues(): array
+    {
+        return Role::toArray();
```

Comment:
30行目に合わせて、`self`の方が良い気がした！
（それか30行目を`Role`にするかで統一した方いいかな）

```suggestion
        return self::toArray();
```